yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -0,0 +1,66 @@
+description: >
+    Interface representing the provisioning state and peer connectivity status
+    of the BMC. It provides properties to indicate whether the BMC has been
+    provisioned and the current connection state with the peer BMC, along with a
+    method to initiate peer provisioning and a signal to report its result.
+
+properties:
+    - name: Provisioned
+      type: boolean
+      flags:
+          - readonly
+      description: >
+          True means the BMC is in a provisioned state.
+    - name: PeerConnected
+      type: enum[self.PeerConnectionStatus]
+      flags:
+          - readonly
+      default: NotDetermined
+      description: >
+          Represents the connection status with the peer BMC.
+
+methods:
+    - name: ProvisionPeer
+      description: >
+          Starts the provisioning process on the peerBmc .
+      parameters:
+          - name: bmcId
+            type: string
+            description: >
+                Identifier of the peer BMC to be provisioned.
+
+signals:
+    - name: PeerProvisioned
+      description: >
+          Emitted when the ProvisionPeer method completes. The signal carries a
```

Comment:
Suggestion: It would be great if you added some information about this signal in the ProvisionPeer method description but I’ll leave it to you.